editor/src/utils/value-parser-utils.ts

```diff
@@ -181,7 +181,7 @@ export function optionalObjectKeyParser<V>(
     if (typeof value === 'object' && !Array.isArray(value) && value != null) {
       const withErrorParser = objectValueParserWithError(parser)
       const valueAsObject: any = value
-      if (key in valueAsObject) {
+      if (key in valueAsObject && valueAsObject[key] !== undefined) {
```

Comment:
I'm not sure how I feel about this, as this implies that it should be passed to the `withErrorParser`.

Reply:
Yeah I feel a bit uneasy about this tbh too. I'm going to tweak the factories so that I can revert this

Reply:
Done